src/TcpServer.php

```diff
@@ -123,6 +124,7 @@
     public function __construct($uri, LoopInterface $loop, array $context = array())
     {
         $this->loop = $loop;
+        $this->context = $context;
```

Comment:
The context for each connection is inherited from the server socket context, is this assignment really needed?